coala_quickstart/Strings.py

```diff
@@ -50,3 +50,18 @@
 
 You can see all of them here: {}
 """.format(BEAR_DOCS_URL)
+
+PRINT_BEARS = {'unusable':
+               {'msg': """
+Based on the configuration options the following bears have been identified
+as unusable (options '-C' or '--ci' (non interactive mode) make the bears that
+need to be configured by the user unusable: don't run in non interactive mode
+or check '--allow-incomplete-sections' for more information on how to include
+the unusable bears):
+""",
+                'colors': ('green', 'red')},
```

Comment:
There seems like an indentation issue in the entire dict, can you please check that?

Reply:
The linter isn't complaining, but indeed it is hard to read. Re-writing it nicely kind of beats me, what I would suggest would be to add variables for each 'msg' value of the dictionary so it should be more "readable". (or another option would be to create variables for the inner dictionaries - 'unusable' and 'usable')
I'm thinking of something like this:
```
PRINT_BEARS = {
                'unusable': {
                    'msg': PRINT_UNUSABLE,
                    'colors': ('green', 'red')
                 },
```

What are your thoughts on this?

Reply:
Yeah, that's a nice idea.